chapters/ch16.md

```diff
@@ -1,70 +1,71 @@
 # 16. Cleaning up mislabeled dev and test set examples
 ->
-
+# 16. Dọn dẹp những mẫu bị gán nhãn nhầm trong tập phát triển và tập kiểm tra
 
 During error analysis, you might notice that some examples in your dev set are mislabeled. When I say “mislabeled” here, I mean that the pictures were already mislabeled by a human labeler even before the algorithm encountered it. I.e., the class label in an example ​(x,y)​ has an incorrect value for ​y.​ For example, perhaps some pictures that are not cats are mislabeled as containing a cat, and vice versa. If you suspect the fraction of mislabeled images is significant, add a category to keep track of the fraction of examples mislabeled:
 ->
+Trong quá trình phân tích lỗi, bạn có thể nhận thấy rằng một vài mẫu trong trong tập phát triển đã bị gán nhãn nhầm. Khi tôi nói "bị gán nhãn nhầm" ở đây, ý tôi là những tấm ảnh đã bị gán nhãn nhầm bởi người dán nhãn trước cả khi thuật toán thuật toán gặp nó. Hay nói cách khác, nhãn lớp của một mẫu (x,y) có giá trị sai cho y.  Ví dụ, có thể một số ảnh không chứa mèo bị gán nhãn nhầm thành có mèo và ngược lại. Nếu bạn nghi ngờ rằng tỷ lệ những ảnh bị gán nhãn nhầm là đáng kể, hãy thêm một hạng mục để theo dõi tỷ lệ các mẫu bị gán nhãn nhầm:
```

Comment:
- đề xuất "trước cả khi thuật toán `xử lý` nó", bị lặp từ 'thuật toán'.
> trước cả khi thuật toán ~thuật toán gặp~ nó

- Ý kiến: không nhất thiết phải dịch sát word by word, vì đọc vô khiến câu không được mạch lạc cho lắm.
> nhãn ~lớp~ của một mẫu (x,y) có giá trị sai cho y

Reply:
* Từ "xử lý" ở đây không đúng lắm vì mô hình lúc này chỉ học từ hình ảnh thôi chứ không tác động gì lên tấm ảnh cả.

* Tiếng Anh "class label" có thể dịch là "nhãn" nhưng nếu dịch chung vậy có thể sẽ không phân biệt được với "target label" cho một vấn đề thuộc regression hay từ "label" nói chung. (có thể không chỉ có một giá trị). Nên mình khá phân vân.